client-v2/src/components/FrontsEdit/Front.tsx

```diff
@@ -204,15 +224,23 @@ const mapStateToProps = (state: State, props: FrontPropsBeforeState) => ({
   front: getFront(state, props.id)
 });
 
-const mapDispatchToProps = (dispatch: Dispatch) => ({
-  dispatch,
-  updateArticleFragmentMeta: (id: string, meta: ArticleFragmentMeta) =>
-    dispatch(updateArticleFragmentMeta(id, meta)),
-  selectArticleFragment: (frontId: string, articleFragmentId: string) =>
-    dispatch(editorSelectArticleFragment(frontId, articleFragmentId)),
-  clearArticleFragmentSelection: (frontId: string) =>
-    dispatch(editorClearArticleFragmentSelection(frontId))
-});
+const mapDispatchToProps = (dispatch: Dispatch) => {
+  return {
+    dispatch,
+    updateArticleFragmentMeta: (id: string, meta: ArticleFragmentMeta) =>
+      dispatch(updateArticleFragmentMeta(id, meta)),
+    selectArticleFragment: (frontId: string, articleFragmentId: string) =>
+      dispatch(editorSelectArticleFragment(frontId, articleFragmentId)),
+    clearArticleFragmentSelection: (frontId: string) =>
+      dispatch(editorClearArticleFragmentSelection(frontId)),
+    removeCollectionItem: (parentId: string, uuid: string) => {
+      dispatch(removeGroupArticleFragment(parentId, uuid));
+    },
```

Comment:
We now delegate deletion behaviour to whatever it is rendering our collection items, as a single collection item can now represent an item in either a main or supporting position.